lib/ui/pages/news_list/home_page.dart

```diff
@@ -0,0 +1,97 @@
+import 'package:allthenews/ui/common/util/dimens.dart';
+import 'package:allthenews/ui/common/util/strings.dart';
+import 'package:allthenews/ui/common/widget/primary_text_button.dart';
+import 'package:allthenews/ui/pages/news_list/primary_news/primary_news_list_entity.dart';
+import 'package:allthenews/ui/pages/news_list/primary_news/primary_news_list_view.dart';
+import 'package:allthenews/ui/pages/news_list/secondary_news/secondary_news_list_entity.dart';
+import 'package:allthenews/ui/pages/news_list/news_list_page.dart';
```

Comment:
tu znów zależy od podejścia, ale jeżeli zostanie news_list_page to proponowałbym inną strukturę pakietów mianowicie fajnie jakby home_page był w pakiecie home, pakiet home zawierałby pakiet news i w tym pakiecie byłby news_list_page oraz dwa inne pakiety primary_news i secondary_news. Jeżeli zmieniamy nazwę na secondaryNewsListPage to przeniósłbym odpowiednio do pakietu secondary_news - ale tu zaznaczam, jeżeli primary i secondary traktujemy domenowo to pierwsza z propozycji jest aktualna :D

Reply:
jw